src/wordpress-playground/index.tsx

```diff
@@ -297,3 +299,104 @@ function zipNameToHumanName(zipName) {
 }
 
 main();
+
+async function overwriteFile() {
+	const targetFile = document.getElementById('target-file').value;
+	const targetContent = document.getElementById('target-content').value;
+	console.log(targetFile);
+	console.log(targetContent);
+	await workerThread.writeFile(targetFile, targetContent);
+}
+
+async function generateZip() {
+	await workerThread.run({
+		code: `
+			<?php
+				$zip = new ZipArchive;
+				$res = $zip->open('/wordpress-playground-export.zip', ZipArchive::CREATE);
+				if ($res === TRUE) {
+					$directories = array();
+					$directories[] = '/wordpress/';
```

Comment:
WordPress path is stored in the `DOCROOT` const (declared a little bit higher in the same file):
```suggestion
					$directories[] = DOCROOT . '/';
```

Reply:
I moved the DOCROOT declaration to `config.ts`. How does that look?

Reply:
That looks awesome!